spleeter/options.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# coding: utf8
+
+""" This modules provides spleeter command as well as CLI parsing methods. """
+
+from tempfile import gettempdir
+from os.path import join
+
+from .separator import STFTBackend
+from .audio import Codec
+
+from typer import Argument, Option
+from typer.models import ArgumentInfo, OptionInfo
+
+__email__ = 'spleeter@deezer.com'
+__author__ = 'Deezer Research'
+__license__ = 'MIT License'
+
+AudioInputArgument: ArgumentInfo = Argument(
+    ...,
+    help='List of input audio file path',
+    exists=True,
+    file_okay=True,
```

Comment:
It seems the API has changed here, with removal of the "-i" option for input files.